src/bust.py

```diff
@@ -97,6 +97,10 @@ async def play(self, interaction: Interaction, skip_count: int = 0) -> None:
         """
 
         await interaction.response.defer(ephemeral=True)
+
+        # Update message channel to where command was issued from
```

Comment:
```suggestion
        # Update message channel to where command was issued from
        # (in case `list` was called from a separate/private channel).
```

Just to say why we're doing this.